### ISSUE TITLE
Memoization failed at unknown configuration 

### ISSUE THREAD

**Issue by YongHee-Kim** [marius311/Memoization.jl#14]
I've been using Memoization for my project for a while and It works great. 

But recentely, I've moved into remote work and set up new development environment in my home
And somehow `@memoize` is not working for my package like this. 

https://user-images.githubusercontent.com/7087745/116016559-f19f4c80-a677-11eb-89fc-7d3d7b3eb835.mp4

But simple test example like this is working just fine. 
``` julia
module Foo 
    using Memoization
    
    @memoize function foo(a, b, c)
        a * b * c 
    end
   
end # module
```

So I've gone to my office, and run my package again there. And `@memoize` just working fine with same operating system and julia version (Windows 10 & Julia 1.6.0)

Any guesses why this happens? I'm confused

**Comment by YongHee-Kim** [comment]
After updating Memoization from 1.6 -> 1.8 It's working again.
Thank you.

**Comment by marius311** [comment]
Yes, 0.1.6 accidentally broke something which was fixed in 0.1.7 & 0.1.8, sorry about that. Good to hear its working now.
